images/macos/templates/macOS-12.anka.pkr.hcl

```diff
@@ -266,7 +266,10 @@ build {
       "${path.root}/../scripts/build/install-pypy.sh",
       "${path.root}/../scripts/build/install-pipx-packages.sh",
       "${path.root}/../scripts/build/install-bicep.sh",
-      "${path.root}/../scripts/build/install-codeql-bundle.sh"
+      "${path.root}/../scripts/build/install-codeql-bundle.sh",
+      "${path.root}/../scripts/build/install-ant.sh",
+      "${path.root}/../scripts/build/install-kotlin.sh",
+      "${path.root}/../scripts/build/install-sbt.sh"
```

Comment:
Current implementation is not really flexible. We need to create one more script for each new case.
How about use something like `"${path.root}/../scripts/build/install-compilable-brew-packages.sh"`?

This will use something like this in the script body:
```bash
source ~/utils/utils.sh

compilable_packages=$(get_toolset_value '.brew.compilable_packages[]')
for package in $compilable_packages; do
    echo "Installing $package..."
    brew_smart_install "$package"
done

invoke_tests "Common" "Compiled"
```

And we also should add block `compilable_packages` to the macOS-12 toolset for `brew`:
```json
...
    "brew": {
...
        "compilable_packages": [
            "ant",
            "kotlin",
            "sbt"
        ],
...
```
And then Pester-test should be rounded by declared block then:
```powershell
Describe "Compiled" -Skip:(-not $os.IsMonterey) {
    It "Apache Ant" {
        "ant -version" | Should -ReturnZeroExitCode
    }

    $kotlinPackages = @("kapt", "kotlin", "kotlinc", "kotlinc-jvm", "kotlin-dce-js")

    It "<toolName> is available" -TestCases ($kotlinPackages | ForEach-Object { @{ toolName = $_ } }) {
        "$toolName -version" | Should -ReturnZeroExitCode
    }

    It "sbt" {
        "sbt -version" | Should -ReturnZeroExitCode
    }
}
```